angr/state_plugins/loop_data.py

```diff
@@ -10,18 +10,60 @@
 
 class SimStateLoopData(SimStatePlugin):
     """
-    This class keeps track of loop-related information for states.
+    This class keeps track of loop-related information for states. Note that
+    we have 2 counters for loop iterations (trip counts): the first recording
+    the number of times one of the back edges (or continue edges) of a loop is
+    taken, whereas the second recording the number of times the loop header (or
+    loop entry) is executed. These 2 counters may differ since compilers
+    usually optimize loops hence completely change the loop structure at the
+    binary level.
     """
```

Comment:
At least one person has misunderstood the point of this plugin and assumed that it can do loop tracking on its own without the otie to guide it, so maybe add a note that it's controlled and updated by LoopSeer?